ner_v2/detectors/numeral/number_range/standard_number_range_detector.py

```diff
@@ -178,52 +179,52 @@ def detect_number_range(self, text):
+    def _get_number_range(self, min_part_match, max_part_match, full_match):
         """
         Update number_range_list and original_list by finding entity value of number tag and original text from
         number_detected_map
         Args:
-            number_tag_min (str or None): tagged min number
-            number_tag_max (str or None): tagged max number
-            matched_text (str): text matching regex
+            min_part_match (str or None): tagged min number
+            max_part_match (str or None): tagged max number
+            full_match (str): text matching regex
             number_range_list (list): list containing all number range detected
             original_list (list): list containing original text of values detected
         Returns:
-            number_range_list (list)
-            original_list (list)
+            (tuple): a tuple containing
+                (list): list containing detected numeric text
+                (list): list containing original numeral text
         """
-        number_range_list = number_range_list or []
-        original_list = original_list or []
+        number_range = None
+        original_text = None
 
         entity_value_min, entity_value_max, entity_unit = None, None, None
 
-        if number_tag_min and number_tag_min in self.number_detected_map:
-            entity_dict = self.number_detected_map[number_tag_min][0]
-            entity_value_min = entity_dict[NUMBER_DETECTION_RETURN_DICT_VALUE]
-            entity_unit = entity_dict[NUMBER_DETECTION_RETURN_DICT_UNIT]
+        if min_part_match and min_part_match in self.number_detected_map:
+            entity_dict = self.number_detected_map[min_part_match].entity_value
+            entity_value_min = entity_dict[numeral_constant.NUMBER_DETECTION_RETURN_DICT_VALUE]
+            entity_unit = entity_dict[numeral_constant.NUMBER_DETECTION_RETURN_DICT_UNIT]
 
-        if number_tag_max and number_tag_max in self.number_detected_map:
-            entity_dict = self.number_detected_map[number_tag_max][0]
-            entity_value_max = entity_dict[NUMBER_DETECTION_RETURN_DICT_VALUE]
-            entity_unit = entity_dict[NUMBER_DETECTION_RETURN_DICT_UNIT]
+        if max_part_match and max_part_match in self.number_detected_map:
+            entity_dict = self.number_detected_map[max_part_match].entity_value
+            entity_value_max = entity_dict[numeral_constant.NUMBER_DETECTION_RETURN_DICT_VALUE]
+            entity_unit = entity_dict[numeral_constant.NUMBER_DETECTION_RETURN_DICT_UNIT]
 
         if self.unit_type and \
                 (entity_unit is None or
-                 self.number_detector_object.language_number_detector.units_map[entity_unit].type != self.unit_type):
-            return number_range_list, original_list
+                 self.number_detector.language_number_detector.units_map[entity_unit].type != self.unit_type):
```

Comment:
I think I mentioned it before about making this dict access a function of number_detector itself